Externals.cfg

```diff
@@ -1,5 +1,5 @@
 [cam]
-branch = noresm2_3_develop
+branch = cam_noresm2_3_v1.0.0b
```

Comment:
Is it possible to make a tag in `cam`, so that the reference is static?

Reply:
Good catch! `cam_noresm2_3_v1.0.0b` _is_ a tag, but the `branch =` bit is confusing. I don't think this will affect manage_externals (it always checks out a detached HEAD) but I have submitted this and another suggestion back to @DirkOlivie.

Reply:
I have modified it now.  Externals.cfg contains now :
tag = cam_noresm2_3_v1.0.0b